app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/onboarding/InsufficientConfidenceLevelController.scala

```diff
@@ -91,7 +91,10 @@ class InsufficientConfidenceLevelController @Inject()(
       InsufficientConfidenceLevelController.haveANinoForm
         .bindFromRequest()
         .fold(
-          e => BadRequest(doYouHaveANinoPage(e)),
+          e => {
+            implicit val s: Option[UserType] = request.authenticatedRequest.userType
```

Comment:
to avoid having to explicitly define this implicit every time you could probably stick something like this in the controller package object:
```scala
implicit def requestToUserType(r: RequestWithSessionData[_]): Option[UserType] = 
  r.authenticatedRequest.userType
```

Reply:
Yeah sorry, I left this hanging around - I tried three different designs one of which involved implicit conversions and this is a remnant of that

Reply:
removed